frontend/src/components/common/form/ParentSelect.tsx

```diff
@@ -148,14 +148,14 @@ export const ParentSelect: React.FC<IParentSelect> = ({
                         ? handleMultiSelectHeaderClick(
                             results.filter(x => x.parentId === +parentId),
                           )
-                        : handleMenuHeaderClick(results.find(x => x.value === parentId)!)
+                        : handleMenuHeaderClick(options.find(x => x.value === parentId)!)
                     }
                   >
                     <b style={{ cursor: 'pointer' }}>
                       {/* project statuses has a different way of selecting parent value*/}
                       {field === 'statusId'
                         ? results.find(x => x.parentId === +parentId)?.parent
-                        : results.find(x => x.value === parentId)?.label}
+                        : results.find(x => x.parentId === +parentId)?.parent}
```

Comment:
Was previously looking through the results for the parent; however, the result could have been rendered out. Now correctly looks at parentId and puts the string value as the header. Then when clicked it will browse the options for the value as it will contain it even if it has been rendered out of the results

Reply:
Woops just noticed I can get rid of ternary